partials/time_to_retirement.html

```diff
@@ -15,111 +15,121 @@
             </div>
         </div>
         <div class="row cannot-retire col-xs-12 col-sm-8 col-sm-offset-2" ng-show="retirement.never_retire_shown">
+        	<div class="big-number-bg">
+                <div class="big-number-wrapper">
+                	<i class="fa fa-remove icon-answer"></i>
+                </div>
+         	</div>
             <div class="col-xs-12 no-retire-message">With your current cash flow trajectory, your future passive income will never safely exceed your expenses. <a href="#/income">Try calculating your countdown with a lower expense rate or higher income.</a>
             </div>
         </div>
         <div class="row cannot-retire col-xs-12 col-sm-8 col-sm-offset-2" ng-show="retirement.already_retired_shown">
-            <div class="col-xs-12 no-retire-message">You're already financially independent.</div>
+        	<div class="big-number-bg">
+                <div class="big-number-wrapper">
+                	<i class="fa fa-check icon-answer"></i>
+                </div>
+         	</div>
+            <div class="until-fi">You're already financially independent.</div>
         </div>
     </div>
     <div class="container-fluid">
         <div class="row">
             <div class="graph-section supporting-content">
                 <div class="supporting-content-wrapper">
-                    <div class="col-md-6" id="graph-wrapper">
+                    <div class="col-md-8" id="graph-wrapper">
                         <div id="retirement-graph"></div>
+                        <div id="explain-text">Financial independence is based on <a href="#/about">the 4% Safe Withdrawal Rate</a></div>
                     </div>
-                    <div class="supporting-section-body col-md-6">
+                    <div class="supporting-section-body col-md-4">
                         <div id="expenses-control" class="input-control col-md-offset-3 col-md-5">
-                            <div class="input-group">
-                                <span class="input-group-btn">
-							        <button class="btn btn-default glyphicon-minus" type="button" ng-click="expenses = expenses - 100"></button>
-							    </span>
-                                <input ng-currency type="text" class="form-control expenses-field" ng-model="expenses" />
-                                <span class="input-group-btn">
-							        <button class="btn btn-default glyphicon-plus" type="button" ng-click="expenses = expenses + 100"></button>
-							    </span>
+                            <div class="">
+                            	<div class="">
+						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="expenses = expenses + 100"></button>

```

Comment:
I don't think we need to make directives right now, but we should at least move the function to the controller. Having it do math or any other logic in the view can lead to difficult bugs and doesn't separate properly.